com.pojtinger.felicitas.Multiplex.json

```diff
@@ -0,0 +1,648 @@
+{
+  "app-id": "com.pojtinger.felicitas.Multiplex",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.golang"],
+  "command": "multiplex",
+  "finish-args": [
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=xdg-download:rw",
+    "--filesystem=xdg-videos:rw",
+    "--filesystem=/tmp:rw",
+    "--talk-name=org.freedesktop.Flatpak"
+  ],
+  "modules": [
+    {
+      "name": "app",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p vendor && cp modules.txt vendor/modules.txt",
+        ". /usr/lib/sdk/golang/enable.sh && export GOBIN=\"/app/bin\" && go generate ./... && go install -v -mod=vendor .",
+        "desktop-file-install --dir=/app/share/applications $FLATPAK_ID.desktop",
+        "install -D -m 0644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml",
+        "for icon in 16x16 22x22 24x24 32x32 36x36 48x48 64x64 72x72 96x96 128x128 192x192 256x256 512x512; do install -D -m 0644 docs/icon-${icon}.png /app/share/icons/hicolor/${icon}/apps/$FLATPAK_ID.png; done"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/pojntfx/multiplex.git",
+          "tag": "v0.1.0",
+          "commit": "09b08c412c150bfcc13ddc6d82caf3e10fe8a7d6"
+        },
+        {
+          "type": "archive",
+          "url": "https://proxy.golang.org/github.com/anacrolix/torrent/@v/v1.52.5.zip",
```

Comment:
usually the generated manifest like for go are "included". This make it easier for everyone.

Reply:
Do you have docs on how this is done? I include the vendored modules list (see `modules.txt`), but I don't seem to be able to figure out how to include the sources from a separate file 👀

Reply:
nvm, looks like its possible to just provide a file path instead of a `type: include`! See 94a3be1f512d76bcd40deb8708fe3c38658a4bcc

Reply:
where is `type: include` documented?

Reply:
It isn't. There isn't any docs on how to "include modules", thus the question: https://www.google.com/search?q=include+sources+flatpak+manifest

Reply:
Like here: 

https://docs.flatpak.org/en/latest/manifests.html#shared-modules

Reply:
Oh thanks, that's neat! Guess `s/include/shared/g` is what I was missing.